drivers/input/touchscreen/tsc2007_iio.c

```diff
@@ -6,6 +6,7 @@
 
 #include <linux/i2c.h>
 #include <linux/iio/iio.h>
+#include <linux/input/touchscreen.h>
```

Comment:
Is this needed?

Reply:
it was because of the added property in the tsc2007.h that the iio driver also uses but in that case its I think better to have the header in the common include instead of the .c files respectively